src/cfile/core.py

```diff
@@ -540,3 +540,22 @@ class Block(Sequence):
     """
     A sequence wrapped in braces
     """
+
+
+class ConditionType(Enum):
+    IF = 0
+    ELSE_IF = 1
+    ELSE = 2
+
+
+class Condition(Block):
```

Comment:
Now you're taking shortcuts. Not all if-statements are wrapped in braces.
In C you can declare statements like this: 
```C
if(condition) ++x; else ++y;
```